playground/src/preview/preview.js

```diff
@@ -50,6 +51,13 @@ class Preview {
       });
     });
 
+    events.subscribe(
+      Experiment.EVENT_TOGGLE_EXPERIMENT,
+      (experiment, force) => {
+        this.toggleExperiment(experiment, force);
```

Comment:
Is the function wrapper needed? What about: `this.toggleExperiment.bind(this);`?